packages/devtools_app/lib/src/screens/debugger/debugger_controller.dart

```diff
@@ -42,6 +43,8 @@ class DebuggerController extends DisposableController
   final codeViewController = CodeViewController();
 
   bool _firstDebuggerScreenLoaded = false;
+  final HoverCardTooltipController hoverCardTooltipController =
```

Comment:
nit: remove type on LHS here for consistency with the code 2 lines above.

Reply:
higher level, it is a little odd this is hung off the debuggerController. Maybethis should be just more a separate top level service associated with hovercards not the debugger as we don't really want multiple hover card tips regardless of whether the debugger is present and a single screen better not diverge on which controller it uses for this.